follower/execution/block_data.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package execution
+
+import (
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/mempool/entity"
+)
+
+type BlockData struct {
+	Block       *flow.Block
+	Collections []*entity.CompleteCollection
+	TxResults   []*flow.TransactionResult
+	Events      []*flow.Event
+	TrieUpdates []*ledger.TrieUpdate
```

Comment:
We will have the following changes to this data:

- `Block *flow.Block` will become `Header *flow.Header`.
- There will be a `StateCommitment flow.StateCommitment` field.

Reply:
How can we get the seals, if we no longer have access to the Block payloads?